### PR TITLE
Add delete button to idea boxes

### LINKED ISSUE #11: Add Delete Buttons
A delete button should be visible on all idea cards now that match the comp.

Image

### PR DESCRIPTION
**Pull request by ben-rosner-williamsburg**
# Description
- Add delete button to idea boxes.
- Change CSS styling of idea boxes to flexbox.

# Contributors
@ben-rosner-williamsburg 

# Checklist
- [x] My PR has an appropriately descriptive and concise title.
- [x] My PR has a clear description of all proposed changes.
- [x] My PR denotes any/all team members who contributed to it.
- [x] My code follows the Turing Style Guides and best practices.
- [x] I ran the code locally and verified that there are no visible errors.
- [x] feat: My code is clean and readable, so my teammates can easily review it.

### CODE REVIEW

**Comment by ericahagle** [comment]
@ben-rosner-williamsburg I know it doesn't strictly state it in the ticket (or spec) other than "that match the comp" but I think those dark purple header sections are probably supposed to be part of the adding of the delete buttons. If you prefer, I could probably fold that into making the favorites buttons, but that might just mean some rejiggering of your work here to make the button fit inside another container.
Screenshot 2023-09-13 at 6 21 42 PM

File: styles.css
Comment (ericahagle) [inline styles.css:121]: @ben-rosner-williamsburg Is there a particular reason you changed this to `display: flex`? I struggled back and forth with this for while and landed on `display: inline-block`, so if you have insight as to why flex works better, I would love the help in understanding it!

**Comment by ericahagle** [comment]
@ben-rosner-williamsburg Just a little organizational polish: Your PR description could use a little love. I see the Notes section there still with the placeholder text. If you don't have anything to add to that section, feel free to delete it! Also, since this is neither a fix nor refactor, you could delete those checkboxes, too.

File: styles.css
Comment (ben-rosner-williamsburg) [reply styles.css:121]: @ericahagle It made positioning the delete button much easier. I'm hesitant to switch back to inline-block for this reason.

Comment (ericahagle) [reply styles.css:121]: Fair! I changed it in the PR I'm working on, too, and it's been peachy for positioning the fave buttons, too. Good call! 🌮

**Comment by ben-rosner-williamsburg** [comment]
> @ben-rosner-williamsburg I know it doesn't strictly state it in the ticket (or spec) other than "that match the comp" but I think those dark purple header sections are probably supposed to be part of the adding of the delete buttons. If you prefer, I could probably fold that into making the favorites buttons, but that might just mean some rejiggering of your work here to make the button fit inside another container. Screenshot 2023-09-13 at 6 21 42 PM

I was able to add this.

File: styles.css
Comment (ericahagle) [inline styles.css]: @ben-rosner-williamsburg Not sure why, but this fella seems to be sticking out of the new idea container. Maybe a smidge more tweaking to get it to fit to the container properly? 
Screenshot 2023-09-14 at 9 16 24 AM

**Review by ericahagle — CHANGES_REQUESTED** [review]
@ben-rosner-williamsburg I also recommend merging the changes from main when you get the chance. I'm fairly certain that you're going to run into some merge conflicts in at least the CSS file and possibly the JS file, too. We had to clean up some stuff that didn't clean up properly during our last battle with merge conflicts, but that's been merged so main should be pristine.
Lmk if you need any help with it!

File: styles.css
Comment (ben-rosner-williamsburg) [reply styles.css]: @ericahagle I will look into that now! Thanks!

Comment (ben-rosner-williamsburg) [reply styles.css]: @ericahagle I just made a new commit here. Let me know if that looks good!

Comment (ericahagle) [reply styles.css]: Much bettah! 🍬 
Screenshot 2023-09-14 at 10 06 22 AM

**Review by ericahagle — APPROVED** [review]
Lgtm! Great job on the button container. We might end up wanting to tweak some sizing things, but I think we should wait until we have the fave buttons merged in, too. Then we can do a little refactor on the whole idea box and its contents, just to make it more polished. 😊 

I think once you get those merge conflicts settled, this should be good to merge into main.

**Pull request merged**
